### DISCUSSION TITLE
Filter katas done with X language

### DISCUSSION THREAD

**Discussion by DanielRomanMartinez** [codewars/codewars.com#2635] [Ideas]
When you are using the the search option you can filter for language, difficult and so on.

I would like to filter by language done or at least show in which language i've completed it. For example in completed kata i can see... 

![image](https://user-images.githubusercontent.com/13299754/158586973-b552159c-42a6-468e-9d21-9e1137f9e139.png)

But when i use the search tool i don't know anything:

![image](https://user-images.githubusercontent.com/13299754/158587099-61315394-67c7-4e57-824e-a449432478e5.png)

I think that some katas are good to practice again with other language to practice sintaxy and so on

**Comment by sofasuit** [comment]
I emphatically second this. Training in multiple languages means you have to click thru to past solutions to see if you've already answered in a given language.

On the Kata's intro page where all available languages are indicated, it would be great to have a visual indicator that indicates a past solution exists in a given language.

**Reply by fikotta** [reply]
^this

**Comment by kazk** [comment]
This was implemented. (Missed this discussion because it's a duplicate of multiple feature requests already filed.)

**Reply by DanielRomanMartinez** [reply]
Happy to see that now works like a charm!!

![image](https://user-images.githubusercontent.com/13299754/185900240-ce51a4eb-c9a7-4083-9aa2-0886763d383b.png)
